app/models/post.rb

```diff
@@ -0,0 +1,10 @@
+class Post < ApplicationRecord
+  after_create :update_posts_counter
+  has_many :comments
+  has_many :likes
+  belongs_to :author, class_name: 'User'
+
+  def update_posts_counter
+    User.find_by(name: author.name).increment!(:posts_counter)
+  end
```

Comment:
- [ ]  Setting associations between models help you call each other from each other so that you don't need to call the class itself. a Post belongs to a user (author). Let's use the related association by calling the method on the author attribute
hint:
```rb
  author.increment!(:posts_counter)
```